src/extension.ts

```diff
@@ -22,6 +22,7 @@ import pollNotifications, {
   cancelNotificationsPolling,
 } from "./notifications/pollNotifications";
 import handleAlpha from "./alphaInstaller";
+import pollMessages, { dispose } from "./statusBarMessages/pollStatusBarMessages";
```

Comment:
It is called a `status bar` so why call the thing `message` and not `status`? It is a bar of statuses...